src/main/java/bio/terra/cli/command/helperclasses/CommandSetup.java

```diff
@@ -0,0 +1,49 @@
+package bio.terra.cli.command.helperclasses;
+
+import bio.terra.cli.auth.AuthenticationManager;
+import bio.terra.cli.context.GlobalContext;
+import bio.terra.cli.context.WorkspaceContext;
+import java.io.PrintStream;
+import java.util.concurrent.Callable;
+
+public abstract class CommandSetup implements Callable<Integer> {
```

Comment:
worth some of the javadoc you had on BaseCommand earlier.
also curious about you're thinking about the rename. If we want all commands to inherit from this, `Base` seems good. If we add much else this to class so that it does something other than `Setup` the current name becomes misleading. Arguably, the PrintStream is not about setup,

Reply:
RE javadoc: Good catch. I meant to copy over the javadoc and forgot. Updated.

RE class name: I don't have strong feelings either way. Happy to change it back to `BaseCommand`. The `PrintStream` stuff should go away in a future PR when I centralize the lookup of stdout and stderr.